src/app/log-tool/data-setup/import-data/import-data.component.ts

```diff
@@ -276,10 +276,10 @@ export class ImportDataComponent implements OnInit {
     this.explorerData.isExample = true;
     this.explorerData = this.logToolDataService.addImportDataSet(exampleDataSet, exampleName, exampleName, this.explorerData);
     this.explorerData = this.logToolDataService.finalizeDataSetup(this.explorerData);
+    this.setExistingDataComplete();
```

Comment:
Good, but use this call order instead:
  await this.logToolDbService.saveData();
  this.runDayTypeAnalysis(true);
  this.setExistingDataComplete();
  this.logToolDataService.explorerData.next(this.explorerData);